tests/layer_tests/pytest.ini

```diff
@@ -3,3 +3,7 @@ markers =
     nightly
     precommit
     timeout
+log_cli = true
```

Comment:
@jiwaszki do you think those settings are needed?

Reply:
@SakuraYM please explain why these setting are required for this PR. I assume that these change MO test suite behavior, do tests still passing without them?

Reply:
sorry, It's not necessary, we can successfully run test without this code.
I'll delete it and submit a new commit

Reply:
> @SakuraYM please explain why these setting are required for this PR. I assume that these change MO test suite behavior, do tests still passing without them?

sorry, It's not necessary, we can successfully run test without this code.
I'll delete it and submit a new commit